index.html

```diff
@@ -18,6 +18,7 @@
         <div id="winner-text"></div>
         <img id="user-image" src="" />
         <div id="username"></div>
+        <ul id="scoreboard" class="list-group"></ul>
```

Comment:
the class `list-group` is causing the issues. It's being used as a selector for populating multiple divs. Change divs to use ids.